components/eam/cime_config/config_compsets.xml

```diff
@@ -271,6 +271,12 @@
     <alias>F-MMF2-ECPP</alias>
     <lname>2000_EAM%MMF2-ECPP_ELM%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV</lname>
   </compset>
+  
+  <!-- MMF compsets for stand-alone single column mode of EAM -->
+  <compset>
+    <alias>F-MMF1-SCAM-ARM97</alias>
+    <lname>AR97_EAM%MMF1-SCAM_ELM%SP_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV</lname>
+  </compset>
```

Comment:
Can you also create the corresponding compset for MMFXX? 
We'll need both for P3 development.

Reply:
sure.